src/simtools/camera/single_photon_electron_spectrum.py

```diff
@@ -119,13 +119,14 @@ def _derive_spectrum_norm_spe(self):
             f"{self.args_dict['simtel_path']}/sim_telarray/bin/norm_spe",
             "-r",
             f"{self.args_dict['step_size']},{self.args_dict['max_amplitude']}",
-            tmp_input_file.name,
         ]
         if tmp_ap_file:
-            command.insert(1, "-a")
-            command.insert(2, f"{tmp_ap_file.name}")
+            command.extend(["-a", f"{tmp_ap_file.name}"])
+            command.extend(["-s", f"{self.args_dict.get('scale_afterpulse_spectrum', 1.0)}"])
```

Comment:
The default is defined here and in the argument parsing itself. Obviously not a problem, but not needed here. (I don't think it means changing the default would mean changing in two places cause I assume the default value here will never be used.)

Reply:
Good point, I removed it here and left it at the parsing stage (required an update in the unit tests)